netbox/circuits/models.py

```diff
@@ -60,7 +60,6 @@ class Provider(ChangeLoggedModel, CustomFieldModel):
 
     tags = TaggableManager()
 
-    serializer = 'circuits.api.serializers.ProviderSerializer'
```

Comment:
I've actually started using the `serializer` attributes on models. `ObjectChangeSerializer` references this to generate a nested representation of related objects when serializing an object for the changelog. Plus I can see it being useful for other things in the future. Let's leave these in place.

Reply:
Nevermind. As you pointed out, this was obsoleted by `get_serializer_for_model()`.